src/generic/listctrl.cpp

```diff
@@ -2230,7 +2253,7 @@ void wxListMainWindow::SendNotify( size_t line,
     // what we're trying to avoid
     if ( !IsVirtual() )
     {
-        if ( line != (size_t)-1 )
+        if ( !IsEmpty() && line != (size_t)-1 )
```

Comment:
This again looks suspicious to me: how can we have a valid `line` if the control is empty?

Reply:
Oops! fixed now !!